### ISSUE TITLE
Vacuum gives errors

### ISSUE THREAD

**Issue by roflmao** [chrislusf/seaweedfs#670]
Forcing gc on master with:

```
 $ curl "http://localhost:9333/vol/vacuum?garbageThreshold=0.15"
```

Volume servers:
```
Jul 02 14:23:45 photol-103 weed[18489]: I0702 14:23:45 18489 volume_vacuum.go:38] Committing vacuuming...
Jul 02 14:23:45 photol-103 weed[18489]: I0702 14:23:45 18489 volume.go:63] Failed to read file size /mnt/data01/data/photo-large-prod-001_4724.dat stat /mnt/data01/data/photo-large-prod-001_4724.dat: use of closed file
Jul 02 14:23:46 photol-103 weed[18489]: I0702 14:23:46 18489 volume_loading.go:83] loading boltdb /mnt/data01/data/photo-large-prod-001_4724.bdb
```

Version: 0.76 (b2b8a00)

Relevant to #601 ?

**Comment by chrislusf** [comment]
Is this on windows?

**Comment by roflmao** [comment]
Linux

**Comment by chrislusf** [comment]
Please use this branch https://github.com/chrislusf/seaweedfs/tree/filer1_maintenance_branch
It has the same fix for #601
